extensions/tachigo-demo-sidepanel/src/styles/fonts.css

```
@@ -0,0 +1,39 @@
+}
+
+@font-face {
+  font-family: Zpix;
+  src: url('../assets/fonts/Zpix.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Zpix CJK;
```

Stylelint (17.6.0)

[error] Expected quotes around "Zpix CJK" (font-family-name-quotes)

(font-family-name-quotes)